Document/0x06g-Testing-Network-Communication.md

```diff
@@ -154,6 +154,7 @@ In general it can be summarised:
 - If connections to 3rd party domains are made (that are not under control of the app owner) it should be evaluated what ATS settings are not supported by the 3rd party domain and if they can be deactivated.
 - If the application opens third party web sites in web views, then from iOS 10 onwards `NSAllowsArbitraryLoadsInWebContent` can be used to disable ATS restrictions for the content loaded in web views
 
+If you want to know whether your domain is actually compatible with ATS, you can use `nscurl --ats-diagnostics <your domain here>` to see how well it is suited for ATS.
```

Comment:
Ok if we use this here https://github.com/OWASP/owasp-mstg/pull/1121 instead?

Reply:
yes